contracts/cw1-whitelist/src/contract.rs

```diff
@@ -27,6 +27,21 @@ impl Cw1 for Cw1WhitelistContract {
         Ok(Response::new())
     }
 
+    fn remove_member(
```

Comment:
`cw1` does not contain `remove_member`. If you want to try msg with multiple alternatives, just create `cw4`